.github/workflows/release.yaml

```
@@ -51,80 +48,56 @@ jobs:
           packages-dir: python/dist/
 
   publish-plugins:
-    needs: publish-core
     name: Create Plugin Releases
     defaults:
       run:
         working-directory: ./python
-    strategy:
-      max-parallel: 1
-      matrix:
-        os: [ubuntu-latest]
-        python-version: ["3.10"]
-        package:
-          [
-            autogen,
-            camel,
-            claude,
-            crew_ai,
-            griptape,
-            julep,
-            langchain,
-            llamaindex,
-            lyzr,
-            openai,
-            praisonai,
-            langgraph,
-            phidata,
-            google,
-          ]
-    runs-on: ${{ matrix.os }}
+    runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
       - name: Set up Python
         uses: actions/setup-python@v5
         with:
-          python-version: ${{ matrix.python-version }}
-      - name: Build ${{ matrix.package }}
+          python-version: "3.10"
+      - name: Build plugins
         run: |
-          pip install twine build
-
-          # Build dist
-          cd plugins/${{ matrix.package }}
-          python -m build
-      - name: Publish ${{ matrix.package }} to PyPI
+          pip install build twine
+
+          plugins="autogen camel claude crew_ai griptape julep langchain llamaindex lyzr openai praisonai langgraph phidata google"
+          mkdir plugins_dist
+          for plugin in $plugins; do
+            cd plugins/$plugin
+            python -m build
```

Consider adding error handling and logging for the plugin build loop. If one plugin fails to build, it could silently fail and continue with others. Adding try-catch with proper error logging would help in debugging issues.